src/app/views/people/people.component.html

```diff
@@ -9,39 +9,40 @@
     </div>
   </div> -->
 
-  <div class="d-flex mt-5" style="margin-left: 25vw;width: 100vw;">
-    <h1 class="float-start">{{getTitleFromType(type)}} Directory</h1>
-  </div>
+  <div class="d-flex mt-5 " style="width: 100vw;">
+    <h1 class="float-start" style="margin-left: 10vw;">{{getTitleFromType(type)}} Directory</h1>
+</div>
+
   <hr class="m-0">
   <!-- Main Display  Starts here -->
   <div class="row justify-content-center mx-5 p-5 mt-0">
     <!-- Main Body Start 3 col division -->
     <div class="col-md-3 card shadow d-none d-md-table-row pb-5 ps-5 " style="width: 20vw;max-height: fit-content;position: sticky; ">
       <h2 class=" my-4 ">Contents</h2>
-      <ol class="facultySideNavHighlight card-body">
+      <ul class="facultySideNavHighlight card-body">
```

Comment:
Explain why use ul here

Reply:
![Screenshot 2024-03-26 165821](https://github.com/ICWaR/icwar.github.io/assets/138218874/e1535991-ab72-4919-aa6d-a66da511801b)


In your case, it was neither showing bullets nor Numbers, but once I did other changes, somehow it started showing Numbers, then I changed it to UL, SS attached for 'ol' case.

Reply:
Also why did you change to ul can you explain that particularly.

Reply:
![Screenshot 2024-03-26 184118](https://github.com/ICWaR/icwar.github.io/assets/138218874/c348d9c3-63ae-4268-88cc-e1d1dcb760c7)

This I achieved after using 'ul'. Prior given SS was the result of 'ol'.